lib/core/element_array_view.cpp

```diff
@@ -45,6 +45,7 @@ ElementArrayViewParams::ElementArrayViewParams(
       m_bucketParams(other.m_bucketParams) {
   expectCanBroadcastFromTo(other.m_iterDims, m_iterDims);
 
+  m_strides.resize(iterDims.ndim());
```

Comment:
I think there should be a constructor for Strides that takes a size. Would need a special tag or a creation function (maybe static member) to disambiguate from a constructor from a single stride.

Reply:
Not sure what the benefit of the complication would be?

Reply:
I'd call it a simplification, because you no longer need to instantiate and resize separately.